ci/cypress.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Do we need 3 versions of this file?

Reply:
No just 2, one for local testing and the other one, for travis (isLocal flag and the videos/screenshots)